repose-aggregator/components/services/http-client-service/http-client-service-impl/src/test/groovy/org/openrepose/core/services/httpclient/impl/HttpConnectionPoolProviderTest.groovy

```diff
@@ -35,12 +35,16 @@ import org.junit.Test
 import org.openrepose.core.service.httpclient.config.HeaderListType
 import org.openrepose.core.service.httpclient.config.HeaderType
 import org.openrepose.core.service.httpclient.config.PoolType
+import org.openrepose.core.services.opentracing.OpenTracingService
 
 import javax.servlet.ServletException
 import javax.servlet.http.HttpServletRequest
 import javax.servlet.http.HttpServletResponse
 import java.nio.charset.Charset
 
+import static org.mockito.Mockito.mock
+
+
 class HttpConnectionPoolProviderTest {
```

Comment:
You didn't add test to prove that you add the interceptors at the appropriate time.